scripts/reuse-existing-aks-cluster.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit # exit immediately if a command exits with a non-zero status.
+set -o nounset # exit when script tries to use undeclared variables.
+set -o pipefail # make the pipeline fail if any command in it fails.
+
+# Color definitions
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+CYAN='\033[0;36m'
+NC='\033[0m' # No Color
+
+# Initialize variables
+KUBE_CONTEXT=""
+
+# Functions to print colored messages
+print_success() {
+    echo -e "${GREEN} $1${NC}" >&2;
+}
+
+print_info() {
+    echo -e "${CYAN} $1${NC}" >&2;
+}
+
+print_warning() {
+    echo -e "${YELLOW} $1${NC}" >&2;
+}
+
+
+print_error() {
+    echo -e "${RED} $1${NC}" >&2;
+}
```

Comment:
Not a blocking comment, but is there a way to reuse the existing functions for this such as in `peer-vnets.sh`?

Reply:
Good thought. We can reuse the functions across bash scripts. I can open up a PR to pull out the common functions.